composer.json

```diff
@@ -32,6 +32,7 @@
         "drupal/core-composer-scaffold": "^9.3",
         "drupal/core-dev": "^9.3",
         "drupal/core-recommended": "^9.3",
+        "drupal/form_options_attributes": "^1.2",
```

Comment:
RW has already an implementation to handle adding option attributes: https://github.com/UN-OCHA/rwint9-site/blob/f563644a643e80963d517beda2094fd58743a57d/html/modules/custom/reliefweb_form/reliefweb_form.module#L29-L30 using the `#option_attributes` property, that do something similar.

Reply:
Module removed